frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.html

```diff
@@ -70,7 +70,7 @@
       <mat-tab label="My Past Challenges ({{ filteredPastChallenges.length }})">
         <div class="challenge-page-title" id = "ongoing-challenges" ><strong class="text-med-black fs-18">My Hosted Challenges</strong></div>
         <div *ngIf="!filteredPastChallenges.length">
-          You haven't hosted any challenge. Please <a [routerLink]="hostTeamsRoutePathCommon" class="highlight-link">click here</a> to host one!
+          You haven't any past hosted challenge.
```

Comment:
Please change the message to `No past hosted challenges found. Please .....`

Reply:
Hey @deshraj I think we shouldn't show the link for hosting a challenge here as hosting a past challenge doesn't make sense.